services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,34 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
-
+  if (xQueueSend(thermalMgrQueueHandle, event, portMAX_DELAY) != pdPASS) {
+    return ERR_CODE_INVALID_QUEUE_MSG;    //IDK WHAT TO SEND HERE
```

Comment:
`xQueueSend` will either return `pdPASS` (which is equivalent to `pdTRUE`) or `errQUEUE_FULL` (look at https://www.freertos.org/a00117.html). So a more appropriate error code to return here would be `ERR_CODE_QUEUE_FULL`